src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -1216,10 +1204,10 @@
           "municipality": "Sison",
           "zip_code": "2434",
           "mayor": {
-            "name": "DANILO C. UY"
+            "name": "ALMA M. LOMIBAO"
           },
           "vice_mayor": {
-            "name": "ALMA M. LOMIBAO"
+            "name": "MOISES 'Jerry' ALAMAY"
```

```suggestion
            "name": "MOISES 'Jerry' P. ALAMAY"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=055&m=41